src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -1,17 +1,27 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="add-notation-dialog">
     <v-card>
-      <v-card-title id="dialog-title">Add a {{displayName}}</v-card-title>
+      <v-card-title id="dialog-title">
+        <span v-if="administrativeDissolution" id="dialog-title"><strong>{{displayName}}</strong></span>
+        <span v-else-if="putBackOn" id="dialog-title"><strong>Correction - {{displayName}}</strong></span>
+        <span v-else id="dialog-title"><strong>Add a {{displayName}}</strong> </span>
+      </v-card-title>
       <v-card-text>
+        <div id="dialog-text" class='dialog-text'>
+          <p v-if="administrativeDissolution || putBackOn"> You are about to put
+          <strong><span class="entity-name">{{getEntityName}}</span>, {{getIdentifier}}</strong>
+          back on the register.</p>
+        </div>
         <div id="notation-text" class="mb-4 mt-2">
-          Enter a {{displayName}} that will appear on the ledger for this entity
+          Enter a {{administrativeDissolution || putBackOn? 'Detail' : displayName}}
```

Comment:
For clarity, please put the compound conditional in parentheses, and add a space before the ? .

Reply:
fixed